packages/cyberstorm/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { MouseEventHandler, ReactNode } from "react";
+import styles from "./componentStyles/Breadcrumb.module.css";
+
+export interface CrumbProps {
+  level: number;
+  path?: string;
+  label: string | ReactNode;
+  onClick?: MouseEventHandler<HTMLAnchorElement>;
+}
+
+export interface BreadcrumbProps {
+  crumbs: CrumbProps[];
+  breadcrumbStyle?: "breadcrumb__default";
+}
+
+// Cyberstorm Breadcrumb component
+
+function generateBreadcrumbs(crumbs: CrumbProps[]) {
+  const sortedArray = crumbs.sort((n1, n2) => {
+    if (n1.level > n2.level) {
+      return 1;
+    }
+    if (n1.level < n2.level) {
+      return -1;
+    }
+    return 0;
+  });
+  return sortedArray.map((crumb, index) => {
+    if (index === 0) {
+      return (
+        <a key={crumb.level} href={crumb.path}>
```

Comment:
Regardless of whether we end up using the linking provider, we probably don't want to use `<a>` here since the main target is Next.js which has its own link component. Using regular links there would cause a full page refresh instead of SPA page transition.